National Library of Norway.js

```diff
@@ -0,0 +1,205 @@
+{
+	"translatorID": "7bd82b70-1ed7-4aaf-bea2-bcbe6429a3ee",
+	"label": "National Library of Norway",
+	"creator": "Dan Michael O. Heggø",
+	"target": "^https?://(www\\.)?nb\\.no/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-05-08 17:30:03"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	National Library of Norway Translator
+	Copyright © 2016 Dan Michael O. Heggø
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.match("/nbsok/search?")) {
```

Comment:
use `indexOf("string") != -1` to test for the presence of literal strings (much more efficient)